warehouse/pom.xml

```diff
@@ -357,7 +357,7 @@
             <dependency>
                 <groupId>org.json</groupId>
                 <artifactId>json</artifactId>
-                <version>20080701</version>
+                <version>20231013</version>
```

Comment:
Lets create a version constant for this one

Reply:
sure